portfolio/src/main/webapp/script.js

```diff
@@ -12,12 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/*
+* Sets a time of non-doing for a function
+*/
+
+
 /**
- * Adds a random greeting to the page.
+ * Gives a random greeting to the page with the interval 3 sec.
  */
+var upd_time = 1500;
```

Comment:
I think this is in milliseconds, so the comment needs to be updated to "every 1.5 seconds".